### DISCUSSION TITLE
Adding a Search Option on the Site

### DISCUSSION THREAD

**Discussion by ssavi-ict** [codinasion/codinasion-programme#1342] [Ideas]
I think adding a **Search Program** feature on the https://codinasion.vercel.app/programme site will be convenient for the users. 
@harshraj8843 will it be possible?

**Comment by harshraj8843** [comment]
Hey @ssavi-ict 
Thanks for the suggestion 😃 
This feature is already tracked in https://github.com/orgs/codinasion/projects/2 and will be added soon. 👍🏻

**Comment by harshraj8843** [comment]
👋🏻  Hey @ssavi-ict 

Please provide your feedback or suggestions regarding [🔍 search bar added !!!](https://codinasion.vercel.app/)

**Comment by ssavi-ict** [comment]
Hey @harshraj8843 
I tried to search the Pangram Search program... 
![image](https://user-images.githubusercontent.com/14172268/171698425-2bee0575-6fc9-4c47-948d-06a46e604127.png)

It returns me nothing.

**Comment by harshraj8843** [comment]
Thanks @ssavi-ict for noticing this issue 👍🏻 

This issue has been fixed 🚀 

<details>
<summary>More about this issue</summary>

## Reasons for this issue

1. Algolia crawler was set to crawl weekly (default)
2. Algolia crawler was not able to crawl shallow route used in `/programme` ( e.g. /programme?page=2 )

## How this issue was resolved

1. Algolia crawler set to crawly daily at `04:00 AM IST` or `22:30 UTC`
2. Added [/sitemap.xml](https://codinasion.vercel.app/sitemap.xml) , so that all pages will be crawled properly
3. Removed shallow routing used in `/programme`, `/tags`, `/dsa`

</details>
